### DISCUSSION TITLE
Mounting Greaseweazle as a virtual disk drive

### DISCUSSION THREAD

**Discussion by m4x10187** [keirf/greaseweazle#291] [General]
I would like to know if it would be possible to create a virtual disk drive on Windows so you can read/write to floppy disks in real time.

**Comment by keirf** [comment]
That would be a "real" disk drive wouldn't it? Anyway: no, you need a normal usb floppy drive for this.

**Reply by m4x10187** [reply]
Thanks for the quick reply, Keir. I was hoping for a solution so I can load copy protected disks into DOSBox like say flux images, I've searched high-and-low for a fork that could hopefully let me do this but had no luck on finding one.

**Comment by LaneaLucy** [comment]
What about Linux? I mean, why buy another thing when the greaseweazle could technically do it and just the not existing software prevents it? A fuse mount would be even enough for most things...

**Reply by keirf** [reply]
A Linux ublk driver would definitely be possible.

**Reply by LaneaLucy** [reply]
That would be really nice. How long would this take? I don't have a greaseweazle yet, but this would be the last nail in the coffin for me to buy one.

**Reply by LaneaLucy** [reply]
i have one now but still no driver...

**Comment by LaneaLucy** [comment]
Why is this closed now?

**Reply by LaneaLucy** [reply]
Thx

**Reply by m4x10187** [reply]
My problem has been solved so I don't have an incentive to reopen. There are tools that I can use, such as DiskImageTool, for converting my own flux dumps into a readable format for 86Box and MartyPC so that I didn't even need to mount the Greaseweazle in the first place. But you are more than welcome to make another issue if you have a related or different problem-I seriously forgot about this issue I made years ago.
